middleware.js

```diff
@@ -9,29 +10,17 @@ export const validateAccessToken = async (req, res, next) => {
 };
 
 export const validateJwtToken = async (req, res, next) => {
-  const token = req.headers["id"];
-
+  const token = req.body.token || req.query.token || req.headers["id"];
   if (!token) {
-    res.status(401).send({ message: " token invalid" });
+    res.status(401).send({ message: " please provide user.id" });
   }
   jwt.verify(token, "secret", (err, user) => {
```

Comment:
I am still seeing. static work "secret" on JWT configuration, use env for this. @mukeshsin

Reply:
ok